extensions/clipyai/src/types.ts

```diff
@@ -0,0 +1,59 @@
+export interface Preferences {
+  apiKey: string;
+  model: string;
+  temperature: string;
+  maxTokens: string;
+  historyLimit: string;
+}
+
```

Comment:
```suggestion
```
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`